amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
You can't enforce test order so I don't thing this is actually testing anything specific?